packages/service-library/src/servicelib/mimetype_constants.py

```diff
@@ -12,6 +12,7 @@
 # NOTE: mimetypes (https://docs.python.org/3/library/mimetypes.html) is already a module in python
 
 MIMETYPE_APPLICATION_JSON: Final[str] = "application/json"
+MIMETYPE_APPLICATION_ND_JSON: Final[str] = "application/x-ndjson"
```

Comment:
what does the "nd" refer to?

Reply:
New line delimited json. That's a format where every line is a json object. Its an extremely nice format e.g. for logging stuff which has to be machine-readable. https://jsonlines.org/
We use it e.g. when streaming the solver logs to Manuel. Each log is wrapped in a json object and streamed down as a single line.